mux/bindings/python/cmux_mux_client/__init__.py

```
@@ -0,0 +1,45 @@
+    IdentifyResult,
+    MuxClient,
+    MuxConnectionError,
+    MuxError,
+    Pane,
+    ProtocolError,
+    ReadScreenResult,
+    Screen,
+    Size,
+    SurfaceResult,
+    Tab,
+    TimeoutError,
```

Ruff (0.15.20)

[error] Import `TimeoutError` is shadowing a Python builtin

(A004)